src/db.py

```diff
@@ -51,6 +51,12 @@ def login(self, username, password):
             else:
                 return False
 
+    def update(self, username, new_email):
+        db = self.mongo.gitsubmit.users
+        user_doc = db.find_one({"username": username})
+        if user_doc is not None:
+            user_doc["email"] = new_email
```

Comment:
In order to update in the database, need this line:
`db.update({"_id": user_doc["_id"]}, user_doc)`